src/validation.js

```diff
@@ -0,0 +1,31 @@
+import Joi from 'joi';
+import _ from 'lodash';
+
+export const schema = Joi.object({
+    login: Joi.string().required(),
+    password: Joi.string().pattern(/\d/).pattern(/[a-zA-Z]/).required(),
+    age: Joi.number().min(4).max(130).required()
+});
+
+const errorResponse = (schemaErrors) => {
+    const errors = schemaErrors.map(error => _.pick(error, ['path', 'message']));
+    return {
+        status: 'failed',
+        errors
+    };
+};
+
+export const validateSchema = (s) => {
```

Comment:
```suggestion
export const validateSchema = (schema) => {
```

avoid non descriptive naming

Reply:
Updated